src/parser/jobs/ast/index.tsx

```diff
@@ -36,6 +36,13 @@ export default new Meta({
 		{user: CONTRIBUTORS.SUSHIROU, role: ROLES.MAINTAINER},
 	],
 	changelog: [
+		{
+			date: new Date('2020-07-13'),
```

Comment:
I think you're a month off. Sentences also have periods but I only care about grammar for @nonowazu I guess.

Reply:
Oh gawd my time machine keeps acting up